be/src/exec/plain_text_builder.cpp

```diff
@@ -37,6 +37,20 @@ Status PlainTextBuilder::init() {
         }
         _converters.emplace_back(std::move(conv));
     }
+
+    if (_options.csv_header)
```

Comment:
Would the header be added more than once if there's more than one chunk?

Reply:
This operation is in the init method and will only be called once for each writer.